ci/azure-pipelines-dev.yml

```diff
@@ -271,7 +271,26 @@ stages:
               pytest -v --postgresql --docs-tests -m integration -k "RUNME_POSTGRES" tests/integration/test_script_runner.py
             displayName: 'Run Flagged Tests'
 
+      - job: runme_script_runner_tests_postgres_msysql
 
+        services:
+          postgres: postgres
+          mysql: mysql
```

Comment:
I'm seeing a test failure - take a look at the mysql stage in this file starting on [line 462](https://github.com/great-expectations/great_expectations/pull/7691/files#diff-8122d09374920cf7c64d1a6ae4ced459d2d631e111612d3ba5b5d091371739f0R462), there are a few other items that are needed to get the database set up, namely [lines 480 thru 491](https://github.com/great-expectations/great_expectations/pull/7691/files#diff-8122d09374920cf7c64d1a6ae4ced459d2d631e111612d3ba5b5d091371739f0R480-R491).